sound/soc/sof/intel/hda-pcm.c

```diff
@@ -118,7 +118,7 @@ int hda_dsp_pcm_hw_params(struct snd_sof_dev *sdev,
 
 	/* set host_period_bytes to 0 if no IPC position */
 	if (hda && hda->no_ipc_position)
-		ipc_params->host_period_bytes = 0;
+		ipc_params->irq_mode = 0;
```

Comment:
Arrh, here we go again. The no_ipc_position has nothing to do with the no_irq mode. we've punted on this and it comes back.

Reply:
@plbossart, the naming wasn't clear to me. Do you think `period_irq` would be OK?